aion/web/elm/src/Update.elm

```diff
@@ -27,7 +28,25 @@ update msg model =
             { model | user = response } ! []
 
         OnQuestionCreated response ->
-            model ! []
+            case response of
+                RemoteData.Success responseData ->
+                    let
+                        oldPanelData =
+                            model.panelData
+
+                        oldCreateQuestionForm =
+                            model.panelData.createQuestionForm
+
+                        newCreateQuestionForm =
+                            Forms.updateFormInput oldCreateQuestionForm "question" ""
+
+                        evenNewerCreateQuestionForm =
```

Comment:
Basically if you define bindings in let area only to use them in the next binding in the same let area, that suggests that you could actually just go ahead and use a pipechain.